pyuvdata/uvdata.py

```diff
@@ -1550,19 +1596,67 @@ def __add__(self, other, run_check=True, check_extra=True,
         ValueError
```

Comment:
I'm not sure what the "right" way to do this is, but you mention that a `ValueError ` is raised in the description of the `phase_center_radec` parameter. I'm not sure if it should also go in this section, or the description above is sufficient. Honestly either is fine with me, more just thinking out loud.

Reply:
sure, I added some text, you can see if it works for you.